migration/data.rst

```diff
@@ -0,0 +1,71 @@
+.. _data_migration:
+
+Data Migration
+==============
+
+Migrating from one database to another has traditionally been difficult. There are two traditional approaches: using a dump and restore that takes downtime, or updating application logic to write to two databases and then switch to the new one.
+
+Now there's an easier way. PostgreSQL 9.4 added logical replication to stream data to a new database even while the existing database continues to sustain application load. It's as if the application automatically writes to two databases rather than one, except with perfect transactional logic.
```

Comment:
`PostgreSQL 9.4 added logical replication` is not correct - Postgres 9.4 added logical *decoding* which can be used for replication, but the database itself does not provided that functionality. Postgres 10 has added actual logical replication into Postgres.